jest.config.packages.js

```diff
@@ -83,10 +89,6 @@ module.exports = {
     // FIXME: For now we do require to build some packages (keyring-api) that is then used
```

Comment:
Is this `FIXME` still relevant?

Reply:
Might not be needed, I'll remove it too.